doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1054 @@
+  * :ref:`http_modem_delta_update_sample` sample, which demonstrates how to add delta modem upgrade support to an application.
+  * :ref:`lib_fmfu_mgmt`, which is a new library that implements parts of the mcumgr management protocol for doing full modem serial updates.
+
+* Updated:
+
+  * :ref:`asset_tracker` application - Updated to use the new FOTA (fota_v2) based on nRF Connect for Cloud.
+  * :ref:`serial_lte_modem` application:
+
+    * Fixed an issue where FOTA downloads were interrupted if an AT command was issued.
+    * Fixed an issue with overflowing HTTP request buffers.
+    * Fixed issues with TCP/UDP server restart.
+    * Added support for allowing only specified TCP/TLS client IP addresses (using the #XTCPFILTER command).
+    * Fixed the unsolicited result code (URC) message format following the 3GPP TS27.007 specification.
+    * Fixed a bug that occurred when the size of an AT command was exactly the maximum buffer size.
+    * Optimized SRAM usage.
+    * Disabled external XTAL usage for UARTE by default.
+    * Added a recovery mechanism for cases where a UART frame error happens.
+    * Enhanced data mode support for TCP/UDP proxy.
+
+  * :ref:`lwm2m_client` sample - Added handling of network failures. The sample now tries to reconnect to the LTE network when a failure is reported.
+  * :ref:`nrf_coap_client_sample` sample - Updated the default server URL to ``californium.eclipseprojects.io``.
+  * :ref:`mqtt_simple_sample` sample - Updated the default server URL to ``mqtt.eclipseprojects.io``.
+  * Extracted the certificate, button, and LED handling functionality from :ref:`http_application_update_sample` sample to :file:`samples/nrf9160/http_update/common`, to share them with :ref:`http_modem_delta_update_sample` sample.
+  * Moved the :ref:`http_application_update_sample` sample from :file:`samples/nrf9160/http_application_update/` to :file:`samples/nrf9160/http_update/application_update`.
+  * :ref:`lib_download_client` library:
+
+    * Reintroduced the optional TCP timeout (enabled by default) on the TCP socket that is used for the download.
+      Upon timeout on a TCP socket, the HTTP download will fail and the ``ETIMEDOUT`` error will be returned via the callback handler.
+    * Added an option to set the hostname for TLS Server Name Indication (SNI) extension.
+      This option is valid only when TLS is enabled.
+
+  * :ref:`lib_date_time` library :
+
+    * Added an API to check if the Date-Time library has obtained a valid date-time. If the function returns false, it implies that the library has not yet obtained valid date-time to base its calculations and time conversions on, and hence other API calls that depend on the internal date-time will fail.
+
+
+  * :ref:`lib_fota_download` - Enabled SNI TLS extension for HTTPS downloads.
```

Comment:
Can you leave the "library" here and fix the title in lib_fota_download instead?

Reply:
Fixed the title in the library.